ftest.sh

```diff
@@ -304,6 +304,18 @@ if ${SETUP_ONLY:-false}; then
     exit 0
 fi
 
+# check if slurm needs to be configured for soak
+
+if [[ $TEST_TAG_ARG == *soak* ]]; then
+    IFS=, read -r -a CONTROL_NODE <<< $TEST_NODES
```

Comment:
Why not use the already parsed node list from lines 55-56?

Reply:
I was going to use nodes but I need to use the first node in the TEST_NODES list.   I know I could use nodes[1] but if  the relation between nodes and TEST_NODES were to change then I may have to change the Control node as well.